### PR TITLE
Milestone 1: Add linters files & Copy the code of a micronaut

### PR DESCRIPTION
**Pull request by mdg0410**
> In this milestone:

- Add linters files
- Copy files of the repository of @barakadanny
- Correct the styles & images paths in the index file

### CODE REVIEW

**Comment by mdg0410** [comment]
Hi @mdg0410, 

You have done a great job so far! 😎🤘
You still need to work on a few issues before moving on to the next project, but you're almost there!



##  Highlight 

✅ The Pull Request is following the GitHub Flow 
⬛ Good UI design  
✅ The commit messages are meaningful 
⬛ The README is well documented 
✅ Linters are working properly 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                    <div class="work__img">
+                        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" rel="noopener noreferrer" class="project_title">Project Title</a>
+                        <img src="./img/work6.jpg" alt="">
+                    </div>
+                </div>
+            
+        </section>
+        <!-- <section id="user-strories">
+            User stories
+        </section> -->
```
Comment (mdg0410) [inline index.html:78]: Remove unnecessary comments from your code, thus improving the presentation of your project.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+                    <box-icon color="white" type='logo' name='twitter'></box-icon>
+
+                </a>
+                <a class="profile__link" >send a mail
+                    <box-icon color="white" name='gmail' type='logo' ></box-icon>
+                </a>
+                <a class="profile__link" >call me
+                    <box-icon color="white" name='phone' type='solid' ></box-icon>
+                </a>
+            </div>
```
Comment (mdg0410) [inline index.html:104]: Your code does not show the social networks section, please check it.

File: index.html (hunk 1 above)
Comment (mdg0410) [reply index.html:78]: Resolved

Comment (PinkMoon25) [reply index.html:78]: - [ ] hi @mdg0410 , good job with the review however please notice as mentioned in the project requirements we should always try to be polite and **"Instead of giving orders, make suggestions. "I think it would be better to use semantic tags" sounds nicer than "Use semantic tags," doesn't it**?" and always try to explain the changes with clear and good reasons

I think we can improve our review if we address the issue like **"I think it will be better if we remove unused code as comments as it will make the codebase clean and will be in accordance with best practices"**

File: index.html (hunk 2 above)
Comment (PinkMoon25) [reply index.html:104]: - [ ] Again as mentioned above we can rephrase this to make it sound better and also help the person with good and clear description i.e. **"Please check your social links as currently they are not visible on the page, I think it might be because you have set the color of the links to white"**

**Review by PinkMoon25 — CHANGES_REQUESTED** [review]
Hi @mdg0410 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark: 
Github review feature :heavy_check_mark: 
Inline comments :heavy_check_mark: 
gitub flow :heavy_check_mark: 
bonus points for using a cool gif :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (mdg0410) [reply index.html:104]: > Correction

You have done a great job with the icons section, however I urge you to check the line containing the icon script and update it. ♻🆙
Once you have done this, you should now be able to display the icons.

File: index.html (hunk 1 above)
Comment (huseyinbiyik) [reply index.html:78]: Kindly improve your review comment as the previous reviewer requested. You can add your improved comment under this line if you wish. 
As mentioned by the previous reviewer, your review needs to have polite language. For example, "Please remove...". 👍

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @mdg0410,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Proper use of inline comment feature 👌

 ## Required Changes ♻️
_Check the comments above the review._

## Optional suggestions
[OPTIONAL]  You can use a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (mdg0410) [reply index.html:78]: > Correction
 
Hi @mdg0410, nice job with your code, however, I kindly suggest you to use semantic tags in your project.
For example, you could start by removing unnecessary comments. 👍

**Comment by mdg0410** [comment]
@huseyinbiyik 

## Thanks bro 😎🤘
